### ISSUE TITLE
Rethink /regen_signs?

### ISSUE THREAD

**Issue by Wuzzy2** [oilboi/Crafter#7]
I saw there's a `/regen_signs` command. I think the problem you want to fix here is that sign entities might die or get removed.

This might interest you:
In MCL2, I handle “node-bound entities” with LBMs. Whenever the chunk is loaded, it checks for nodes like signs, banners, modspawners, etc. If they don't have an entity, it is respawned.

It's not a perfect solution, but good enough. I know that `/clearobjects` is quite aggressive in removing entities.

**Comment by oilboi** [comment]
Yes that's a good idea, it's going to have to take a complete redesign though

**Comment by oilboi** [comment]
This is still an extremely difficult issue, I will look at this more later
